src/main/java/frc/robot/utilities/EncoderConversions.java

```diff
@@ -78,6 +78,10 @@ public static double magToDegrees(double magTick) {
         return magToDegrees(magTick, 1);
     }
 
+    /**
+     * @param degrees the angle wanted to convert to ticks
+     * @return the angle as ticks
```

Comment:
@ param degrees angle of the wheel or gear in degrees
@ param gearRatio  Gear Ratio between Mag and Mechanism (set to 1 for Mag RPM)
@ return the angle of the Mag in ticks